compiler/env/OMRVMEnv.hpp

```diff
@@ -108,6 +108,33 @@ class OMR_EXTENSIBLE VMEnv
    bool ensureOSRBufferSize(TR::Compilation *comp, uintptrj_t osrFrameSizeInBytes, uintptrj_t osrScratchBufferSizeInBytes, uintptrj_t osrStackFrameSizeInBytes) { return false; }
    uintptrj_t thisThreadGetOSRReturnAddressOffset(TR::Compilation *comp) { return 0; }
 
+   /**
+    * @brief Returns offset from the current thread to the intermediate result field.
+    * The field contains intermediate result from the latest guarded load during concurrent scavenge.
+    */
+   uintptrj_t thisThreadGetGSIntermediateResultOffset(TR::Compilation *comp) { return 0; }
+   /**
+    * @brief Returns offset from the current thread to the flags to check if concurrent scavenge is active
+    */
+   uintptrj_t thisThreadGetConcurrentScavengeActiveByteAddressOffset(TR::Compilation *comp) { return 0; }
+   /**
+    * @brief Returns offset from the current thread to field with the base address of the evacuate memory region
+    */
+   uintptrj_t thisThreadGetEvacuateBaseAddressOffset(TR::Compilation *comp) { return 0; }
+   /**
+    * Returns offset from the current thread to field with the top address of the evacuate memory region
+    */
+   uintptrj_t thisThreadGetEvacuateTopAddressOffset(TR::Compilation *comp) { return 0; }
+   /**
+    * Returns offset from the current thread to the operand address field.
+    * It contains data from the most recent guarded load during concurrent scavenge
+    */
+   uintptrj_t thisThreadGetGSOperandAddressOffset(TR::Compilation *comp) { return 0; }
```

Comment:
Could we add brief tags to the above two definitions so Doxygen properly picks them up? Thanks for making these changes by the way!